packages/glab.portable/tools/chocolateyinstall.ps1

```diff
@@ -1,16 +1,16 @@
-﻿$ErrorActionPreference = 'Stop'; # stop on all errors
+$ErrorActionPreference = 'Stop'; # stop on all errors
 $toolsDir   = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
-$fileLocation = Join-Path $toolsDir 'glab_1.22.0_Windows_i386.zip'
-$fileLocation64 = Join-Path $toolsDir 'glab_1.22.0_Windows_x86_64.zip'
+$url = "https://gitlab.com/gitlab-org/cli/-/releases/v1.24.1/downloads/glab_1.24.1_Windows_i386.zip"
+$url64 = "https://gitlab.com/gitlab-org/cli/-/releases/v1.24.1/downloads/glab_1.24.1_Windows_x86_64.zip"
 
 $packageArgs = @{
   packageName   = $env:ChocolateyPackageName
   unzipLocation = $toolsDir
-  file         = $fileLocation
-  file64         = $fileLocation64
-  checksum      = '43e86b78e919ec0a1947157c1eeaf62fa8bc03087f09affb28293af939ac4389'
+  url           = $url
+  url64bit      = $url64
+  checksum      = '85e9b4ee96337ebe48b040a552b2cb7327be86d235ff6e2c9d555be32c9fe56c'
   checksumType  = 'sha256'
-  checksum64    = '1b07931cbd9547fdf33bc5c636a5e70bc01d7ef68768cff13f9655623ce411d8'
+  checksum64    = '6b553a556ba7011a0c9fcf8a5f947ab60c6aabfaada47391ea7ba5703b08392b'
   checksumType64= 'sha256'
 }
-Install-ChocolateyZipPackage @packageArgs # https://docs.chocolatey.org/en-us/create/functions/install-chocolateyzippackage
\ No newline at end of file
+Install-ChocolateyZipPackage @packageArgs # https://docs.chocolatey.org/en-us/create/functions/install-chocolateyzippackage

```

Comment:
```suggestion
Install-ChocolateyZipPackage @packageArgs # https://docs.chocolatey.org/en-us/create/functions/install-chocolateyzippackage

# Remove any zip files from previous installs
Remove-Item $toolsDir/*.zip
```

The `Install-ChocolateyZipPackage` doesn't remove the zip files, so they could be left behind between versions. This just makes sure the 1.22.0 files are removed.